src/assertions/toHaveCompletedExecutionWithStatus/index.ts

```diff
@@ -8,11 +8,13 @@ export default {
     stateMachineName: string,
     expectedStatus: string
   ): Promise<TestResultOutput> {
-    const stepFunctions = new AWS.StepFunctions();
-    const stepFunctionsObject =  await StepFunctions.build();
+    const stepFunctions: AWS.StepFunctions = new AWS.StepFunctions();
```

Comment:
Would it be better to use the `AWSClient` object exported from `helpers/general` here, in case the user has e.g. set the region using the CLI flags?